docs/index.rst

```diff
@@ -9,9 +9,9 @@ for developing on the edX platform.
 Requirements
 ------------
 
-* Python (2.7, 3.5, 3.6)
-* Django (1.8, 1.9, 1.10, 1.11)
-* Django REST Framework (3.2+)
+* Python (3.5, 3.6, 3.8)
+* Django (2.2, 3.2)
+* Django REST Framework (3.9+)
```

Comment:
@xitij2000 
Can we be sure that this upgrade won't break any dependent library? I'm just thinking that this is a breaking change and might need a thorough testing in all the environments (3.5, 3.6, 3.8).

The following two places can be changed accordingly:

https://github.com/edx/edx-drf-extensions/blob/17a7e337b1aa142fff336dd5fca68399b129b9ba/tox.ini#L2

https://github.com/edx/edx-drf-extensions/blob/17a7e337b1aa142fff336dd5fca68399b129b9ba/.github/workflows/ci.yml#L16

Reply:
I just removed the versions that I knew were definitely not supported anymore. I will reduce it to just the ones tested by the CI.